cypress/e2e/token-details.test.ts

```diff
@@ -1,4 +1,6 @@
-import { getTestSelector } from '../utils'
+import { getClassContainsSelector, getTestSelector } from '../utils'
+
+const UNI_GOERLI = '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984'
```

Comment:
You should pull this from src/constants/addresses.ts

Reply:
this results in a build error in the test, so i'm leaving as is for now